pages/press-room/2023-10-26-program-launch.tsx

```diff
@@ -0,0 +1,97 @@
+import { ReactElement } from 'react';
+import { Typography } from '@material-ui/core';
+
+import MediaRelease from '../../modules/press-room/MediaRelease';
+
+const singleMediaRelease = (): ReactElement => {
+  return (
+    <MediaRelease
+      title="Digital Waves NL Launches Impact-Driven Technology Learning Experience to Newfoundland and Labrador Youth"
+      subtitle="Participants learn coding and design skills to help develop Quadrangle NL&rsquo;s groundbreaking 2SLGBTQIA+ knowledge base app."
+      body={(
+        <>
+          <img
+            src="../../assets/images/press/2023-10-26-program-launch.png"
+            alt="Learn to code for community impact."
+          />
+          <Typography variant="caption" component="p">
+            Image by&nbsp;
+            <a
+              target="_blank"
+              rel="noreferrer"
+              href="https://www.freepik.com/free-photo/close-up-smiley-friends-with-laptop_18492461.htm#query=teens%20laptop&position=2&from_view=search&track=ais"
+            >
+              Freepik
+            </a>
+          </Typography>
+
+          <Typography>
+            <strong>St. John&rsquo;s, NL [October 25, 2023]</strong> -&nbsp;
+            <a href="https://digitalwavesnl.ca" target="_blank" rel="noreferrer">Digital Waves NL</a>, a non-profit organization dedicated to
```

Comment:
You know, my first thought here was using relative links would be nicer, but upon a bit of quick google, absolute links are better for SEO so I think they may make more sense even though this is slightly more awkward to test on local dev.

Reply:
Yeah I think it _may_ be important in this context only in case people copy/paste for news articles